src/modules/users/components/GasStation/TransactionCard/TransactionStatus.tsx

```diff
@@ -70,7 +75,7 @@ const TransactionStatus = ({
           <FormattedMessage
             {...MSG.transactionState}
             values={{
-              status,
+              status: TRANSACTION_STATUSES.READY,
```

Comment:
I don't understand this change. Does this mean that from now on the status will always be set to `TRANSACTION_STATUSES.READY` ?

Reply:
Oof, my bad! This is a leftover from when I was testing the changes. 

Thank you for finding it! I thought I deleted them all.